packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/deeply-nested-asyncs/actual.js

```diff
@@ -0,0 +1,13 @@
+async function s(x) {
+  let t = async (y) => {
+    let r = async (z) =>  {
+      await z;
+      return this.x;
+    }
+    await r;

```

Comment:
It's very syntactic and not-for-real-use example. I didn't change behaviour of Babel helpers `asyncToGenerator`. As I know, it's valid situation to yield regular values. `asyncToGenerator` helper resolved value or did `Promise.resolve(value)`.